contracts/System.sol

```diff
@@ -18,6 +18,7 @@ contract System {
   uint8 constant public GOV_CHANNELID = 0x09;
   uint8 constant public SLASH_CHANNELID = 0x0b;
   uint8 constant public CROSS_STAKE_CHANNELID = 0x10;
+  uint8 constant public BC_FUSION_CHANNELID = 0x11; // new channel id for cross-chain stake migration after BC-fusion
```

Comment:
where do it register `BC_FUSION_CHANNELID ` ?

Reply:
By governance